src/components/messenger/list/conversation-list-panel.tsx

```diff
@@ -9,24 +9,40 @@ import { IconButton } from '../../icon-button';
 import { ConversationItem } from './conversation-item';
 import { InviteDialogContainer } from '../../invite-dialog/container';
 import { Button, Modal } from '@zero-tech/zui/components';
+import { Item, Option } from '../lib/types';
+import { UserSearchResults } from './user-search-results';
+import { itemToOption } from '../lib/utils';
 
 export interface Properties {
   conversations: Channel[];
 
-  onConversationClick: (conversationId: string) => void;
+  search: (input: string) => any;
   startConversation: () => void;
+  onCreateConversation: (userId: string) => void;
+  onConversationClick: (conversationId: string) => void;
 }
 
 interface State {
   filter: string;
   inviteDialogOpen: boolean;
+  userSearchResults: Option[];
 }
 
 export class ConversationListPanel extends React.Component<Properties, State> {
-  state = { filter: '', inviteDialogOpen: false };
+  state = { filter: '', inviteDialogOpen: false, userSearchResults: [] };
 
-  searchChanged = (search: string) => {
+  searchChanged = async (search: string) => {
     this.setState({ filter: search });
```

Comment:
Curious, does this cause a rerender before we hit the `await` for the api search? would be nice to quickly filter the existing convos so that you're not stuck waiting for a slow api request.